### ISSUE TITLE
where is setUserData method?

### ISSUE THREAD

**Issue by shadow0162** [imkira/unity-countly#4]
i found other SDK(iOS or Android) have a method to set user data, like:nickname,gender,email,organization.
how can do the same thing in Unity sdk?

**Comment by gorkem-cetin** [comment]
Corresponding documentation is here: 

http://resources.count.ly/v1.0/docs/countly-sdk-for-android

Basically you can send a custom key-value to the server so this special keys are populated in User Profiles.

**Comment by shadow0162** [comment]
Yeah, I konw the setUserData in Android SDK.I just not find the same method in Unity SDK.

``` C#
CountlyManager.Init("def31073c215e19abe9e0ae5f0a9fa08c7a59ff0");
CountlyManager.Emit()
//can use?
CountlyManager.SetUserData()
```

**Comment by gorkem-cetin** [comment]
I see :) 

Unity SDK is developed by Mario. This is a community driven SDK. You can ask him or open a ticket here: 

https://github.com/imkira/unity-countly

**Comment by shadow0162** [comment]
So it seems that we need to extend the Unity SDK.
